### PR TITLE
Add logo and rename pieshop

### PR DESCRIPTION
**Pull request by devnook**

### CODE REVIEW

File: src/client/manifest.json [hunk 1]
```diff
@@ -3,22 +3,22 @@
   "short_name": "SHOP",
```
Comment (rowan-m) [inline src/client/manifest.json]: Change to `PWA Shop` for consistency?

File: src/templates/partials/header.hbs [hunk 2]
```diff
@@ -21,7 +21,10 @@
     </div>
   </menu>
 
-  <h1><a href="/" title="Link to home page">pieshop</a></h1>
+  <h1 id="logo"><a href="/" title="Link to home page"><svg viewBox="0 0 24 24">
```
Comment (rowan-m) [inline src/templates/partials/header.hbs]: The logo is causing the title to conflict with the search box on smaller screens.

---

`master`
screenshot 2018-06-29 at 16 14 51

---

`logo`
screenshot 2018-06-29 at 16 15 42

Comment (samdutton) [reply src/templates/partials/header.hbs]: We can fix this, but we should avoid displaying both the logo and name at smaller viewport sizes.

**Comment by samdutton** [comment]
Almost all big e-commerce sites use a logotype (i.e. text) not a purely graphical logo(mark).

Also — If we have to have a logo, we should make sure it doesn't look like an icon (e.g. by colouring it).

Overall, I wonder whether we need a logo at all, and whether text might be better.

**Pull request merged**